spec/projection_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'check_graphite/projection'
+require 'nagios_check'
+
+describe describe CheckGraphite::Projection do
+  let :check do
+    Class.new do
+      attr_reader :values
+      include NagiosCheck
+      include CheckGraphite::Projection
+    end.new
+  end
+
+  describe 'primary value' do
```

Comment:
I read "describe should be used for classes and methods and context for sections within describes that require extra qualification" as a mockist argument. To me `describe` is about a unit of functionality and a particular method can be split into any number of units. The current labels are atrocious tho. Fixed.